CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - ESLint - Fix `search-builder` according to eslint rules.
 - ESLint - Fix eslintrc to allow dangle underscore.
 - ESLint - Fix `records-decorator` according to eslint rules.
+- Filter - Fix or filter when filtering on a belongsTo with another simple attribute.
```

Comment:
```
- Filters - Fix "OR" filter when filtering on a reference field along with another simple attribute condition.
```

Reply:
What about charts? Isn't there any fix to do?

Reply:
Yeah it should be the same fix. I got to know first if it's good for you here then I'll try to refacto this part.